src/DatePicker/index.js

```diff
@@ -65,13 +69,14 @@ export default class DatePicker {
 
     this.ui.datePicker.className = 'date-picker'
     this.ui.datePicker.innerHTML = `
-        <button type="button" class="date-picker__toggle" aria-label="Toggle Date Picker">Toggle Date Picker</button>
+        <button type="button" id="date-picker__toggle" aria-label="Toggle Date Picker" aria-controls="date-picker__calendar">Toggle Date Picker</button>
 
-        <div class="date-picker__calendar" style="display: none">
-          <p class="calendar__header" role="heading"></p>
-          <button id="calendar__prev" type="button">${this.props.previousLabel}</button>
-          <button id="calendar__next" type="button">${this.props.nextLabel}</button>
+        <div id="date-picker__calendar" aria-labelledby="calendar__header" aria-dialog="true" aria-hidden="true" style="display: none">
```

Comment:
Is this ID unique enough?  Is there a way you could iterate over each calendar you initialize and make it `date-picker__calendar-1`, 2, 3, etc?